### PR TITLE
switching back to projectblacklight/blacklight-marc

### PR DESCRIPTION
**Pull request by banukutlu**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ gem 'rsolr', '>= 1.0'
 gem 'webpacker', '~> 3.5'
 
 gem 'blacklight', '>= 7.0.0.rc1', github: 'projectblacklight/blacklight'
-gem 'blacklight-marc', github: 'psu-libraries/blacklight-marc'
+gem 'blacklight-marc', github: 'projectblacklight/blacklight-marc'
```
[the remark this answers is not in the document]
Comment (banukutlu) [reply Gemfile]: what about pegging to `7.0.0.alpha`?

[the remark this answers is not in the document]
Comment (cdmo) [reply Gemfile]: if that existed I'd say yes 😅

[the remark this answers is not in the document]
Comment (banukutlu) [reply Gemfile]: yeah right not yet.  But i'd guess there will be likely one when BL7.0 releases so until then I think I am fine with leaving both bl and bl marc as is and peg both to a release when BL7.0 releases 🤔  but yes going with master is not sth we would want to continue doing. what do you think?

[the remark this answers is not in the document]
Comment (banukutlu) [reply Gemfile]: I can put an issue to change both later

[the remark this answers is not in the document]
Comment (cdmo) [reply Gemfile]: works for me!

**Pull request merged**